### PR TITLE
feat: guest order with steps

### LINKED ISSUE #481: I want to place a guest order
to refine

### PR DESCRIPTION
**Pull request by patzick**
## Changes

closes #481 

- [x] separated most of the logic from components
- [x] storing checkout data in 15 min cache
- [ ] create issues for logged-in user components

Changed structure: 
image

image

### CODE REVIEW

File: packages/default-theme/components/checkout/steps/OrderReviewStep.vue [hunk 1]
```diff
@@ -0,0 +1,144 @@
+<template>
+  <div>
+    <SfHeading
+      title="4. Order review"
+      class="sf-heading--left sf-heading--no-underline title"
+    />
+    <SfAccordion first-open class="accordion mobile-only">
+      <SfAccordionItem header="Personal Details">
+        <PersonalDetailsSummary @click:edit="$emit('click:edit', 0)" />
```
Comment (mkucmus) [inline packages/default-theme/components/checkout/steps/OrderReviewStep.vue:9]: you can use constants as you did in PersonalDetailsSummary:
```@click="$emit('click:edit', CHECKOUT_STEPS.PERSONAL_DETAILS)```

File: packages/default-theme/layouts/default.vue [hunk 2]
```diff
@@ -30,33 +30,38 @@ export default {
...
+    SwPluginTopNavigation,
   },
   computed: {
     componentBreadcrumbs() {
       // TODO probably move to vuex now as it's not rendered on server side
       return (
-        this.$route.matched.map(r => {
-          return r.components.default.options.data().breadcrumbs
-        })[0] || {}
+        this.$route.matched
```
Comment (mkucmus) [inline packages/default-theme/layouts/default.vue:39]: maybe some kind of helper for such a case?

File: packages/default-theme/components/checkout/steps/OrderReviewStep.vue (hunk 1 above)
Comment (patzick) [reply packages/default-theme/components/checkout/steps/OrderReviewStep.vue:9]: yeah, I planned to create an issue for that :)

File: packages/default-theme/layouts/default.vue (hunk 2 above)
Comment (patzick) [reply packages/default-theme/layouts/default.vue:39]: definitely, but not with this issue, here I just covered a case when it sometimes showed an error

File: packages/default-theme/components/checkout/steps/OrderReviewStep.vue (hunk 1 above)
Comment (patzick) [reply packages/default-theme/components/checkout/steps/OrderReviewStep.vue:9]: https://github.com/DivanteLtd/shopware-pwa/issues/610

**Comment by patzick** [comment]
@mkucmus added issues for all TODOs for checkout https://github.com/DivanteLtd/shopware-pwa/labels/Checkout

**Pull request merged**